### PR TITLE
Document Edge Function Request.url changes

### PR DESCRIPTION
**Pull request by agutekanst-llnw**
Documentation for Request URL changes to the Edge Function `Request` class in v7.13.6

### CODE REVIEW

File: guides/applications/v7/edge_functions.md [hunk 1]
```diff
@@ -211,9 +211,21 @@ Edge functions are passed a `Request` instance representing the incoming request
   - **JSON**: `await request.json()`
   - **Text**: `await request.text()`
 - **Method**: `request.method` to get the HTTP method of the request.
-- **URL**: `request.url` provides the full URL, and `request.path` gives the request path.
+- **URL**:
+  - `request.url`: Provides the full URL.
+  - `request.path`: Provides the request path.
+  - `request.originalUrl`: Provides the original URL sent by the user-agent.
```
Comment (jsernaque) [inline guides/applications/v7/edge_functions.md:217]:
```suggestion
  - `request.originalUrl`: Requires {{ PRODUCT }} version X.X.X or later. Provides the original URL sent by the user-agent. This property is only present the incoming request provided to the `handleHttpRequest` handler.
```

File: guides/applications/v7/edge_functions.md [hunk 2]
```diff
@@ -211,9 +211,21 @@ Edge functions are passed a `Request` instance representing the incoming request
...
+    - Only present on the incoming request provided to the `handleHttpRequest` handler on vX.X.X+.
```
Comment (jsernaque) [inline guides/applications/v7/edge_functions.md:218]: Merge this line with the one above it.

File: guides/applications/v7/edge_functions.md [hunk 3]
```diff
@@ -211,9 +211,21 @@ Edge functions are passed a `Request` instance representing the incoming request
...
 - **Cloning**: To clone a request without its body, use `request.cloneWithoutBody()`.
 
+<Callout type="important">
```
Comment (jsernaque) [inline guides/applications/v7/edge_functions.md:221]: Move this callout to the URL list item.
Most edge functions do not require CDN-as-code. 
```suggestion
  <Important>
  As of {{ PRODUCT }} version X.X.X, the `request.path` and `request.url` properties return the [rewritten](/applications/performance/rules/features#rewrite-url) path or URL, respectively. 
  </Important>
```

File: guides/applications/v7/edge_functions.md [hunk 4]
```diff
@@ -211,9 +211,21 @@ Edge functions are passed a `Request` instance representing the incoming request
...
+
+As of vX.X.X, the behavior of `request.path` and `request.url` has changed to provide the URL/path with any router-configured rewrite rules applied.
+
+The original URL sent by the user-agent is now available at `request.originalUrl` on the incoming `Request` instance provided to the `handleHttpRequest` handler.
```
Comment (jsernaque) [inline guides/applications/v7/edge_functions.md:225]: Is this line needed? It seems repetitive.

**Review by jsernaque — COMMENTED** [review]
Minor suggestions. Feel free to contact me to discuss.

File: guides/applications/v7/edge_functions.md (hunk 4 above)
Comment (agutekanst-llnw) [reply guides/applications/v7/edge_functions.md:225]: Good point!

File: guides/applications/v7/edge_functions.md [hunk 5]
```diff
@@ -214,18 +214,12 @@ Edge functions are passed a `Request` instance representing the incoming request
 - **URL**:
   - `request.url`: Provides the full URL.
   - `request.path`: Provides the request path.
-  - `request.originalUrl`: Provides the original URL sent by the user-agent.
-    - Only present on the incoming request provided to the `handleHttpRequest` handler on vX.X.X+.
+  - `request.originalUrl`: Requires {{ PRODUCT }} version 7.13.6 or later. Provides the original URL sent by the user-agent. This property is only present the incoming request provided to the `handleHttpRequest` handler.
```
Comment (jsernaque) [inline guides/applications/v7/edge_functions.md:217]:
```suggestion
  - `request.originalUrl`: Requires {{ PRODUCT }} version 7.13.6 or later. Provides the original URL sent by the user-agent. This property is only present when the incoming request is provided to the `handleHttpRequest` handler.
```

File: guides/applications/v7/edge_functions.md [hunk 6]
```diff
@@ -214,18 +214,12 @@ Edge functions are passed a `Request` instance representing the incoming request
...
+  <Important>
+	  As of {{ PRODUCT }} version 7.13.6, the `request.path` and `request.url` properties return the [rewritten](/applications/performance/rules/features#rewrite-url) path or URL, respectively.
```
Comment (jsernaque) [inline guides/applications/v7/edge_functions.md:219]:
```suggestion
  As of {{ PRODUCT }} version 7.13.6, the `request.path` and `request.url` properties return the [rewritten](/applications/performance/rules/features#rewrite-url) path or URL, respectively. 
```

File: guides/applications/v7/edge_functions.md (hunk 5 above)
Comment (agutekanst-llnw) [reply guides/applications/v7/edge_functions.md]: Good catch! I missed a word. I meant this to read:
"This property is only present **[on]** the incoming request provided to the `handleHttpRequest` handler."

We always provide a request to the `handleHttpRequest` handler, so hopefully this is a bit more clear. I think the other way might cause confusion/imply that there are cases when the edge function wouldn't when be provided a request.

**Pull request merged**